frontend/src/components/pages/connect/overview.tsx

```diff
@@ -74,15 +82,38 @@ const getDefaultView = (defaultView: string): { initialTab: ConnectView; redpand
   }
 };
 
-const WrapUseSearchParamsHook: FunctionComponent<{ matchedPath: string }> = (props) => {
+const WrapKafkaConnectOverview: FunctionComponent<{ matchedPath: string }> = (props) => {
   const { search } = useLocation();
   const searchParams = new URLSearchParams(search);
   const defaultTab = searchParams.get('defaultTab') || '';
-  return <KafkaConnectOverview defaultView={defaultTab} {...props} />;
+
+  const { data, isLoading } = useQuery<KafkaConnectors>({
+    queryKey: ['kafka-connect-connectors'],
+    queryFn: async () => {
+      const response = await config.fetch(`${config.restBasePath}/kafka-connect/connectors`, {
+        method: 'GET',
+        headers: [['Content-Type', 'application/json']],
+      });
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch kafka connect connectors');
+      }
+
+      return response.json();
+    },
+  });
```

Comment:
Let's use a dedicated custom React hook for this, maybe call it `kafka-connect.tsx`